### PR TITLE
Cancel earlier queued benchmarks

### PR DESCRIPTION
**Pull request by pepeiborra**
Same trick as used for the test workflow

### CODE REVIEW

**Review by Ailrun — APPROVED** [review]
Awesome, thanks!

File: .github/workflows/bench.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@ jobs:
         os: [ubuntu-latest]
 
     steps:
+      # Cancel queued workflows from earlier commits in this branch
+      - uses: fkirc/skip-duplicate-actions@master
+
```
Comment (Ailrun) [inline .github/workflows/bench.yml:17]: Maybe this indentation breaks the actions?

Comment (jneira) [reply .github/workflows/bench.yml:17]: Mmm it shouldn't afaik, yaml lets you do it

Comment (Ailrun) [reply .github/workflows/bench.yml:17]: It looks like inconsistent indentation does not work in YAML. The following lines have less indentation.

**Pull request merged**